### PR TITLE
Removed TextFormatting from some files

### PR DESCRIPTION
**Pull request by TrapinchO**
Files edited:
- modules/utilities/overlays/OverlayEvents.java
-- The only place where it is not possible was a method "onTerritoryWar"
- modules/utilities/overlays/ui/GearViewerUI.java
-- it already had the static import

### CODE REVIEW

**Comment by TrapinchO** [comment]
Other modules that have a lot of colors should I remove it there too?
https://github.com/Wynntils/Wynntils/blob/development/src/main/java/com/wynntils/modules/utilities/overlays/inventories/RarityColorOverlay.java
- 37 finds

https://github.com/Wynntils/Wynntils/blob/development/src/main/java/com/wynntils/modules/utilities/events/ClientEvents.java
- 45 finds

https://github.com/Wynntils/Wynntils/blob/development/src/main/java/com/wynntils/modules/chat/overlays/gui/TabGUI.java
- 82 finds
- done

EDIT: Should I remove it from these too?

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java [hunk 1]
```diff
@@ -48,6 +48,8 @@
 
     private static String totemName;
 
+    boolean isVanished = false; // used in onEffectApplied
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: Does this have to be package-private?

File: src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java [hunk 2]
```diff
@@ -48,31 +48,30 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.NBTTagInt;
 import net.minecraft.util.ResourceLocation;
-import net.minecraft.util.text.TextFormatting;
 
 public class GearViewerUI extends FakeGuiContainer {
-
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java:54]: Did you run a formater?

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: It is not used anywhere else.

It is a variable I created when fixing the vanish timer bug. Do you want it public?

File: src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java (hunk 2 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java:54]: no

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (HighCrit) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: No, private

File: src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java (hunk 2 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/ui/GearViewerUI.java:54]: Or at least I don't remember doing so

File: src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java (hunk 1 above)
Comment (TrapinchO) [reply src/main/java/com/wynntils/modules/utilities/overlays/OverlayEvents.java]: done

**Comment by TrapinchO** [comment]
Please someone notice it

**Comment by HeyZeer0** [comment]
Make sure to disable git autocrlf for your next PR @TrapinchO

**Comment by TrapinchO** [comment]
> Make sure to disable git autocrlf for your next PR @TrapinchO

Will do! (when/if I find it)

**Pull request merged**